app/views/search/index.html.erb

```diff
@@ -134,7 +134,8 @@
                   <div class="vcard">
                     <p class="address adr">
                       <span class="visually-hidden fn org"><%= firm.name %></span>
-                      <span class="address__line street-address"><%= firm.address_line_one %></span>
+                      <span class="address__line line-one"><%= firm.address_line_one %></span>
+                      <span class="address__line line-two"><%= firm.address_line_two %></span>
```

Comment:
This isn't mandatory right? Perhaps we should hide this if there is no value. Otherwise there'll be a blank line.